contracts/shared/helpers/Fixture.ts

```diff
@@ -159,17 +160,27 @@ export default class Fixture {
     );
   }
 
+  async createBLSWallet(): Promise<BlsWalletWrapper> {
+    return BlsWalletWrapper.connect(
+      `0x${Math.floor(Math.random() * 0xffffffff).toString(16)}`,
```

Comment:
Prefer non-random/deterministically created private keys to aid in test reproducibility for triaging/debugging.